.github/workflows/build-image.yaml

```diff
@@ -6,8 +6,6 @@ on:
   push:
     branches:
       - main
-    tags:
-      - "*"
```

Comment:
I think we need this, as that's how we release, no? @didierofrivia 
tags could be only `v[0-9]+\.[0-9]+\.[0-9]+(-.*)?` or something tho